pkg/csv/writer.go

```diff
@@ -39,6 +39,11 @@ func (c *GenericWriter) AddValue(value Value) {
 	c.values = append(c.values, value)
 }
 
+// Append to value
+func (c *GenericWriter) AppendValue(value Value, field string) {
```

Comment:
nit: Rename this to `AppendToValue`  to avoid confusion with existing `AddValue` function.